static/src/javascripts/projects/common/modules/ui/last-modified.js

```diff
@@ -4,14 +4,20 @@ import fastdom from 'lib/fastdom-promise';
 
 const lastModified = (): void => {
     fastdom
-        .read(() => document.getElementsByClassName('js-lm')[0])
-        .then(jsLm => {
+        .read(() => ({
+            jsLm: document.getElementsByClassName('js-lm')[0],
```

Comment:
Is it worth giving these vars less abbreviated names?

Reply:
Definitely think thats a good idea. I personally don't find it clear at all (just trying to restore the functionality)